Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -490,18 +488,31 @@ void AzAssetBrowserWindow::CreateToolsMenu()
         m_toolsMenu->addAction(collapseAllAction);
 
         m_toolsMenu->addSeparator();
-        auto* projectSourceAssets = new QAction(tr("Hide Product Assets"), this);
+        auto* projectSourceAssets = new QAction(tr("Hide Engine Folders"), this);
         projectSourceAssets->setCheckable(true);
         projectSourceAssets->setChecked(true);
         connect(projectSourceAssets, &QAction::triggered, this,
             [this, projectSourceAssets]
             {
-                m_ui->m_thumbnailView->HideProductAssets(projectSourceAssets->isChecked());
                 m_ui->m_searchWidget->ToggleProjectSourceAssetFilter(projectSourceAssets->isChecked());
             });
         m_toolsMenu->addAction(projectSourceAssets);
-
         m_ui->m_searchWidget->ToggleProjectSourceAssetFilter(projectSourceAssets->isChecked());
+
+        auto* unusableProductAssets = new QAction(tr("Hide Unusable Product Assets"), this);
+        unusableProductAssets->setCheckable(true);
+        unusableProductAssets->setChecked(true);
+        connect(
+            unusableProductAssets,
+            &QAction::triggered,
+            this,
+            [this, unusableProductAssets]
+            {
+                m_ui->m_searchWidget->ToggleUnusableProductsFilter(unusableProductAssets->isChecked());
+            });
+        m_toolsMenu->addAction(unusableProductAssets);
+        m_ui->m_searchWidget->ToggleUnusableProductsFilter(unusableProductAssets->isChecked());
```

Comment:
This is OK for now, but we should do a follow-up to migrate these actions to the new action manager.